rundeckapp/grails-app/migrations/core/ScheduledExecution.groovy

```diff
@@ -227,6 +227,14 @@ databaseChangeLog = {
 
     changeSet(author: "rundeckuser (generated)", failOnError:"false", id: "3.4.0-22", dbms: "h2") {
         comment { 'rename filter to "FILTER' }
+        preConditions(onFail: 'MARK_RAN') {
```

Comment:
I think you may need this same code for the execution table. I'm getting a failure on it regarding the "filter" column not being found.